README.md

```diff
@@ -11,17 +13,20 @@ Pytorch implementation for reproducing COCO results in the paper [StackGAN: Text
 
 
 ### Dependencies
-python 2.7
+python 2.7 or higher
 
-Pytorch
+Base Deeplearning Framework: `Pytorch`
 
 In addition, please add the project folder to PYTHONPATH and `pip install` the following packages:
 - `tensorboard`
+- `tensorboardX`
 - `python-dateutil`
 - `easydict`
 - `pandas`
 - `torchfile`
 
+**Or** use the following command `pip install -r requirements.txt`
```

Comment:
Guidance to install the dependencies using a single command as I have introduced a `requirements.txt` file